packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -454,7 +454,15 @@ function formatOutputs(outputs) {
 }
 
 function showEvents(events) {
-  events = events.sort((a, b) => new Date(a.Timestamp) > new Date(b.Timestamp));
+  events = events.sort((a, b) => {
```

Comment:
I would not use sort based on timestamp because CFN already sorts the events for us, we just get it in reverse order, and if we get two events with the same timestamp we can hit an issue that we'll still print it in wrong order. Please update to just reverse the entries and it will be ok

Reply:
Thank you for your review! That is good to know. As that sounds to require domain knowledge, I left that as a comment in the implementation.